definitions/infra-awsrekognition/golden_metrics.yml

```diff
@@ -0,0 +1,36 @@
+requests:
+  title: Total Request Count
+  unit: COUNT
+  queries:
+    aws:
+      select: sum(aws.rekognition.CallCount)
+      from: Metric
+      eventId: entity.guid
+      eventName: entity.name
+userError:
+  title: User Error count
+  unit: COUNT
+  queries:
+    aws:
+      select: sum(aws.rekognition.UserErrorCount)
+      from: Metric
+      eventId: entity.guid
+      eventName: entity.name
+runtimeUserErrors:
+  title: Server Error Count
```

Comment:
For consistency, the word "Count" should be lowercase

Reply:
adderssed